repo_health/check_dependabot.py

```diff
@@ -0,0 +1,56 @@
+"""
+Checks to make suree dependabot files are there and they have enough ecosystems
```

Comment:
Can we rephrase this sentence to something like this, because these check files are just for analyzing the health of the repo?

```suggestion
Checks to identify whether the Dependabot file exists and which ecosystems are covered under Dependabot
```

Reply:
yea, make sense to me